src/transformers/integrations/ggml.py

```diff
@@ -490,11 +509,24 @@ def converted(self) -> Tokenizer:
         return tokenizer
 
 
+class GGUFBloomConverter(GPT2Converter):
+    def __init__(self, tokenizer_dict):
+        self.original_tokenizer = GGUFTokenizerSkeleton(tokenizer_dict)
```

Comment:
This seems to be missing a superclass `__init__()` call, is that on purpose?

Reply:
We don't need initializer from a base class, as we don't provide any arguments like original_tokenizer, we just call converted method